src/ast/mod.rs

```diff
@@ -3403,6 +3447,10 @@ pub enum Statement {
         /// Cursor name
         name: Ident,
         direction: FetchDirection,
+        /// Differentiate between dialects that fetch `FROM` vs fetch `IN`
+        ///
+        /// [MsSql](https://learn.microsoft.com/en-us/sql/t-sql/language-elements/fetch-transact-sql)
+        from_or_in: AttachedToken,
```

Comment:
Not sure what's best here, it could also be two separate Optional<AttachedToken> fields

Reply:
Could we represent it with an explicit enum?
e.g
```rust
enum FetchPosition {
    From
    In
}
```

Reply:
Done 👍